src/components/hooks/useUser.js

```diff
@@ -163,20 +163,26 @@ function useUserImpl() {
     isValidating,
   ]);
 
+  const hasUserParameters = Object.keys(loanerInfo?.userParameters).length > 0;
+
   //TODO give diffferent name
-  const isGuestUser =
-    !isAuthenticated && Object.keys(loanerInfo?.userParameters).length > 0;
+  const isGuestUser = !isAuthenticated && hasUserParameters;
 
   return {
     authUser: userData?.user || {},
     isLoading: userIsLoading,
     error: userDataError,
+    // User is loggedIn (verified through adgangsplatformen)
     isAuthenticated,
+    // User exist in culr
     hasCulrUniqueId,
+    // User has a CPR verified account in culr
     isCPRValidated,
-    loanerInfo,
+    // User is loggedIn as guest user
```

Comment:
kommentaren er måske lidt misvisende. måske "user cannot be verified, but userParameters are saved in session"